src/app/teams/teams.component.html

```diff
@@ -27,17 +27,25 @@
       <mat-cell *matCellDef="let element">
         <ng-container [ngSwitch]="element.userStatus">
           <button *ngSwitchCase="'member'" mat-raised-button color="primary" class="margin-lr-3" (click)="toggleMembership(element.doc, true)">
-            Leave
+            <mat-icon class="margin-lr-3">clear</mat-icon>
+            <span i18n>Leave</span>
           </button>
           <button *ngSwitchCase="'unrelated'" mat-raised-button color="primary" class="margin-lr-3" (click)="requestToJoin(element.doc)">
-            <ng-container i18n>Request to Join</ng-container>
+            <mat-icon class="margin-lr-3">group_add</mat-icon>
+            <span i18n>Request to Join</span>
           </button>
           <button *ngSwitchCase="'requesting'" mat-raised-button color="primary" disabled>
```

Comment:
can we also add icon here even though its disabled so it looks consistent. It is displayed as button